src/__stories__/config.js

```diff
@@ -3,7 +3,7 @@ const { configure } = require("@kadira/storybook")
 function loadStories() {
   require("../components/__stories__/artwork")
   require("../components/__stories__/artwork_grid")
-  require("../components/__stories__/login_header")
+  require("../components/__stories__/modal_header")
```

Comment:
I had looked into this, the issue is that this file gets evaluated in both Node and the browser and thus globbing a FS at that point isn’t that straightforward.

However, FuseBox does have an in-browser API that we can use to glob, so maybe we should conditionally use either the `node-glob` package or the FuseBox API.